spec/features/publish_interface/viewing_provider_users.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Viewing provider users" do
+  before do
+    given_i_am_authenticated_as_a_provider_user
+    when_i_visit_the_provider_users_page
+  end
+
+  scenario "i can view associated users" do
+    then_i_can_view_users_associated_with_the_provider
+  end
```

Comment:
Minor style observation — worth merging the before block into the scenario. Nothing gained in terms of readability by separating them.